test/gen-pgbouncer-conf.bats

```diff
@@ -34,7 +34,8 @@ teardown_file() {
     assert_success
     cat "$PGBOUNCER_CONFIG_DIR/pgbouncer.ini"
     assert_line 'Setting DATABASE_URL_PGBOUNCER config var'
-    assert grep "server_tls_sslmode = prefer" "$PGBOUNCER_CONFIG_DIR/pgbouncer.ini"
+    assert grep "auth_type = scram-sha-256" "$PGBOUNCER_CONFIG_DIR/pgbouncer.ini"
+    assert grep "server_tls_sslmode = require" "$PGBOUNCER_CONFIG_DIR/pgbouncer.ini"
     assert grep "db1= host=host dbname=name?query port=5432" "$PGBOUNCER_CONFIG_DIR/pgbouncer.ini"
     assert grep "db2= host=host2 dbname=dbname port=7777" "$PGBOUNCER_CONFIG_DIR/pgbouncer.ini"
     assert grep "user" "$PGBOUNCER_CONFIG_DIR/users.txt"
```

Comment:
(optional) Should these `users.txt` assertions check the password is present after the username too, now it's being written directly? 

eg `assert grep "user pass" "$PGBOUNCER_CONFIG_DIR/users.txt"`

(And for bonus points updating `user2`'s password to `pass2` to verify the correct passwords are being written for each user?)

Reply:
Yep, added!